src/clj_gatling/simulation.clj

```diff
@@ -16,16 +16,20 @@
 (defn- now [] (System/currentTimeMillis))
 
 (defn asynchronize [f ctx]
-  (let [parse-response (fn [result]
-                         (if (vector? result)
-                           {:result (first result) :end-time (now) :context (second result)}
-                           {:result result :end-time (now) :context ctx}))]
+  (let [parse-result   (fn [result]
+                         (if (instance? Exception result)
```

Comment:
A lot of `Throwable`s we probably don't want to catch, but `AssertionError` is a `Throwable`, not an `Exception`, and people may wish to use `(assert ...` in their steps. What do you think?